modules/custom/dkan_api/src/Controller/Dataset.php

```diff
@@ -2,24 +2,49 @@
 
 namespace Drupal\dkan_api\Controller;
 
-use Drupal\dkan_api\Storage\DrupalNodeDataset;
-use Drupal\dkan_schema\SchemaRetriever;
-use JsonSchemaProvider\Provider;
+use Symfony\Component\DependencyInjection\ContainerInterface;
 
+/**
+ * Dataset.
+ */
 class Dataset extends Api {
 
-  private $nodeDataset;
+  use \Drupal\dkan_common\Util\TestableParentCallTrait;
```

Comment:
Lets take the "Testable" portion from the name of this trait.

Reply:
fixed in 2bc6d19308e68af785992490ca1db753f0e9e8fb